frontend/src/pages/forms/index.tsx

```diff
@@ -29,6 +29,11 @@ const navCardItems: readonly NavigationCardItem[] = [
     header: 'Dev Portfolio Assignments',
     description: 'Submit opened and reviewed pull requests.',
     link: '/forms/devPortfolio'
+  },
```

Comment:
Can you gate this navigation card on a feature flag here? i.e. add a line
```
export const ENABLE_COFFEE_CHAT = false;
```
https://github.com/cornell-dti/idol/blob/5bb5d22ce0c0d1680bb39a0e83e9bb86c24ae697/frontend/src/consts.ts

there's an adminOnly attribute that you can assign here, which will hide the card for non-admins. This way, we won't mislead anyone while we're developing this month, and when we're ready to launch we can just flip the flag to true.